### ISSUE TITLE
Merge master branch back into develop

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster#968]
Today in the release procedure we have steps to merge the develop branch into the master branch. Then we tag the master branch with the new version number. After that the release procedure is done from a Git perspective.

A consequence of this is that the version tags are excluded from the develop branch. This in turn means that `git describe --tags` gets tripped up. Having a working `git describe --tags` would be helpful when testing different development versions of our packages.

**Comment by matsduf** [comment]
Do you already have a suggestion for those steps?

**Comment by mattias-p** [comment]
Maybe we could add something like this to the end of "Merge develop branch into master"?

```sh
git checkout origin/develop
git merge origin/master
git push --force origin HEAD:develop
```

I haven't tested these particular commands though.

**Comment by matsduf** [comment]
> Maybe we could add something like this to the end of "Merge develop branch into master"?

I guess it must be a separate step after the release. It should be OK.

**Comment by mattias-p** [comment]
Yeah, it's probably more robust to make it a separate step and put it somewhere after the release tag is created.

**Comment by matsduf** [comment]
As far as I can see this is already resolved by https://github.com/zonemaster/zonemaster/blob/master/docs/internal-documentation/maintenance/ReleaseProcess-release.md#18-merge-master-into-develop. Can we close this issue?
